src/components/planning/Planning.vue

```diff
@@ -233,6 +228,9 @@ export default {
     async getDistanceMatrices () {
       this.distanceMatrices = await distanceMatrix.list();
     },
+    updatePlanningHeaderHeight () {
+      setTimeout(() => { this.planningHeaderHeight = this.$refs['planningHeader'].clientHeight; }, 100);
```

Comment:
Tu peux directement écrire `this.$refs.planningHeader.clientHeight`

Reply:
non ca ne fonctionne pas ...

Reply:
🆗